services/manage/app/controllers/manage/orders/edit/lineitems/edit/ads/edit.js

```diff
@@ -1,15 +1,23 @@
 import Controller from '@ember/controller';
 import { ObjectQueryManager } from 'ember-apollo-client';
 import ActionMixin from '@base-cms/parcel-plug-manage/mixins/action-mixin';
+import { computed } from '@ember/object';
 
 import adName from '@base-cms/parcel-plug-manage/gql/mutations/ad/name';
 import adImage from '@base-cms/parcel-plug-manage/gql/mutations/ad/image';
 import adWidth from '@base-cms/parcel-plug-manage/gql/mutations/ad/width';
 import adHeight from '@base-cms/parcel-plug-manage/gql/mutations/ad/height';
 import adUrl from '@base-cms/parcel-plug-manage/gql/mutations/ad/url';
 import deleteAd from '@base-cms/parcel-plug-manage/gql/mutations/ad/delete';
+import pauseAd from '@base-cms/parcel-plug-manage/gql/mutations/ad/pause';
 
 export default Controller.extend(ActionMixin, ObjectQueryManager, {
+  isActive: computed.not('model.paused'),
+  isPausedDisabled: computed('isActionRunning', 'model.status', function() {
+    if (this.get('isActionRunning')) return true;
```

Comment:
Disabling of the switch should just use `disabled={{isActionRunning}}` in the template... or, rather, the autosave switch component should disable itself when running. Then you do not need to use this